data/publications/lahti.json

```diff
@@ -1,4 +1,131 @@
 [
```

Comment:
I hope this was automatically generated?

Reply:
yes